lib/index.js

```diff
@@ -11,9 +11,11 @@ var cache = new Map();
  * @param {Object} obj
  * @param {string} key
  * @param {!Function} method
+ * @param {!Function} extend -  { getter(), setter() }
  */
-var method = module.exports = (obj, key, method) => {
-  method = method === undefined ? () => {} : method;
+var method = module.exports = (obj, key, method, extend) => {
```

Comment:
Thinking that `method` and `extend` arguments could be combined, so that muk could be called like

```js
muk(obj, 'a', 2);
```

and

```js
muk(obj, 'a', {
  getter: () => ...
  setter: () => ...
}
```

Without setting `method` to null.